<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamerun</class>
 <widget class="QDialog" name="gamerun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="stocksTable">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>901</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buySellBtn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>510</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter # of Shares</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buySubmitBtn">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sellSubmitBtn">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>560</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Sell</string>
   </property>
  </widget>
  <widget class="QLabel" name="buySellLbl">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>520</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stockDropdown">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>510</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Stock</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="cashBalanceLbl">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="profitsLbl">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skipBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Skip To Next Day</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentDayLbl">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cash Balance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Profits:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Day:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveGameBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Game</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
